### PR TITLE
[236] Evaluation Forms list view feedback

### LINKED ISSUE #236: Evaluation Forms list view feedback
# What/Why
Feedback on #60:
1. Add the **Evaluation Period** column to the desktop and mobile layouts as shown in the mockup.
2. Remove the **Delete** button from the list view. This is no longer part of the product requirements or design.

### Figma Link
Image
Image

## Acceptance Criteria
- [x] column is present and displays the correct dates for each form in the list
  - [x] closing_date is required for the form
  - [x] start_date is taken from the phase like it is on the show/create view

_Disregard the following if this work **does not** include any UI changes_
- [x] WAVE screenshot
- [ ] Automated a11y tests (axe-rspec specs)
- [x] Keyboard accessibility (tab navigation, form interactions)
- [x] Responsiveness at desktop and mobile

### PR DESCRIPTION
**Pull request by cpreisinger**
#236

### CODE REVIEW

File: app/helpers/evaluation_forms_helper.rb [hunk 1]
```diff
@@ -9,6 +9,13 @@ def challenge_phase_title(challenge, phase)
     "#{challenge.title} - Phase #{phase_number(phase)}"
   end
 
+  def evaluation_period(evaluation_form)
+    start_date = evaluation_form.closing_date.strftime("%m/%d/%Y")
```
Comment (stepchud) [inline app/helpers/evaluation_forms_helper.rb]: phase.end_date
```suggestion
    start_date = evaluation_form.pase.end_date.strftime("%m/%d/%Y")
```

File: app/views/evaluation_forms/_table.html.erb [hunk 2]
```diff
@@ -13,31 +14,29 @@
...
-            </td>
+          </th>
+
+          <td data-label="Assigned to Challenge:">
+            <% if evaluation_form.challenge %>
+              <%= challenge_with_phase(evaluation_form) %>
+            <% end %>  
+          </td>
+
+          <td data-label="Assigned to Challenge:">
```
Comment (stepchud) [inline app/views/evaluation_forms/_table.html.erb]: Change label
```suggestion
          <td data-label="Evaluation Period:">
```
Screenshot 2024-11-01 at 11 51 34 AM

**Review by stepchud — CHANGES_REQUESTED** [review]
couple fixes, can you add the WAVE report screenshot too?

**Comment by cpreisinger** [comment]
Screenshot 2024-11-01 at 3 27 21 PM

**Review by stepchud — APPROVED** [review]
looks good to me

**Pull request merged**